src/measure_performance_MEAN_py3.py

```diff
@@ -53,7 +53,11 @@ def ssh_exec(ssh_client, cmd):
 def quilt_ssh(ip):
 	client = paramiko.SSHClient()
 	client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+<<<<<<< HEAD
+	client.connect(ip, username="quilt", password="SSH_PASSWORD")
```

Comment:
Why do you need SSH_PASSWORD? It should only require your private key to connect to the remote server right?